winui/Kanban/getting-started.md

```diff
@@ -57,18 +57,55 @@ public sealed partial class MainWindow : Window
 
 ## Populate WinUI Kanban item source
```

Comment:
Add this content

This section explains how to populate the WinUI Kanban control's `ItemSource` by creating and binding both default and custom task data models.